trello_utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import requests
+
+TRELLO_KEY = os.environ.get('TRELLO_KEY')
+TRELLO_TKN = os.environ.get('TRELLO_TKN')
+
+TRELLO_BOARD = '5f48fa1737fa77134ffe1e17'
+TRELLO_TODO_LIST = '5f48fa17b70abe80f7f0942d'
+TRELLO_DONE_LIST = '5f48fa17d5f15e744c86a725'
+TRELLO_URL_BASE = 'https://api.trello.com/1/'
+
+CARD_DONE_STATUS = 'Done!'
+CARD_TODO_STATUS = 'To-do'
+
+LOCAL_CARDS = []
+
+
+def trimCardsList(trello_cards):
+    for card in trello_cards:
+        updateOrAddCard(Card(
+            card['id'],
+            card['name'],
+            getCardStatus(card['id'])
+        ))
+    print LOCAL_CARDS.sort(key=cardComparator)
```

Comment:
This print statement is sneakily changing the value of LOCAL_CARDS. If you delete this line, the result of the function is no longer being sorted, which is surprising - we normally want to delete print statements before merging if you included it to help with debugging.

You should do the .sort() on a separate line.